bitmovin-analytics-datasource/src/types/queryFilter.ts

```diff
@@ -9,10 +9,20 @@ export type QueryFilterOperator = (typeof QUERY_FILTER_OPERATORS)[number];
 export const SELECTABLE_QUERY_FILTER_OPERATORS: Array<SelectableValue<QueryFilterOperator>> =
   QUERY_FILTER_OPERATORS.map((o) => ({ value: o, label: o }));
 
-export type QueryFilter = {
+/** This type is needed because of legacy reasons.
+ * In the angular plugin the value was saved as a string in a dashboard JSON file. */
+export type InputQueryFilter = {
   name: QueryAdAttribute | QueryAttribute;
   operator: QueryFilterOperator;
-  value: QueryFilterValue;
+  value: string;
 };
 
-export type QueryFilterValue = boolean | number | string | string[] | null;
+/** QueryFilter type with the correct value type that is accepted by the Bitmovin API */
+export type OutputQueryFilter = {
```

Comment:
rename, because `convertFilterValueToProperType` util creates it , so it would make more sense
```suggestion
export type ProperTypedQueryFilter = {
```